examples/intel/intel_examples/domain/domain_reputation.py

```diff
@@ -14,16 +15,13 @@
 intel = DomainIntel(token, config=config)
 
 
-def main():
+def main() -> None:
     print("Checking domain...")
-
-    try:
```

Comment:
Why remove try/catch in just some examples?

Reply:
I removed them when they did nothing more than `print(e)`, since that doesn't seem any more helpful than just letting the exception bubble up.